controll/controller.js

```diff
@@ -0,0 +1,14 @@
+const { app } = require("../app")
```

Comment:
You don't need to require in app as you are not using it. Please make sure not to require anything you don't need in